From: Hilko Bengen <bengen@debian.org>
Date: Fri, 21 Oct 2011 23:46:24 +0200
Subject: Fix --{en,dis}able-install-daemon

This fixes a

    configure: error: conditional "INSTALL_DAEMON" was never defined.

when specifying --disable-daemon
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index e4e25b8..b8512b1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -263,9 +263,9 @@ if test "x$enable_daemon" = "xyes"; then
                   [enable installing the daemon under $sbindir @<:@default=no@:>@])],
                 [],
                 [enable_install_daemon=no])
-        AM_CONDITIONAL([INSTALL_DAEMON],[test "x$enable_install_daemon" = "xyes"])
         AC_MSG_RESULT([$enable_install_daemon])
 fi
+AM_CONDITIONAL([INSTALL_DAEMON],[test "x$enable_install_daemon" = "xyes"])
 
 dnl Build the appliance?
 AC_MSG_CHECKING([if we should build the appliance])
-- 
